src/main/java/org/folio/service/BaseService.java

```diff
@@ -200,19 +198,18 @@ public CompletableFuture<Void> handlePutRequest(String endpoint, JsonObject reco
   public CompletableFuture<Void> handleDeleteRequest(String endpoint, HttpClientInterface httpClient,
       Map<String, String> okapiHeaders, Logger logger) {
     CompletableFuture<Void> future = new CompletableFuture<>();
-    logger.debug(CALLING_ENDPOINT_MSG, HttpMethod.DELETE, endpoint);
+    logger.debug("Trying to delete object with endpoint: {}", endpoint);
     try {
       httpClient.request(HttpMethod.DELETE, endpoint, okapiHeaders)
         .thenAccept(this::verifyResponse)
         .thenApply(future::complete)
         .exceptionally(t -> {
-          String errorMessage = String.format(EXCEPTION_CALLING_ENDPOINT_MSG, HttpMethod.DELETE, endpoint);
-          logger.error(errorMessage, t);
+          logger.error("Object cannot be deleted with using endpoint: {}", endpoint, t);
           future.completeExceptionally(t);
           return null;
         });
     } catch (Exception e) {
-      logger.error(String.format(EXCEPTION_CALLING_ENDPOINT_MSG, HttpMethod.DELETE, endpoint), e);
+      logger.debug("Error deleting object by endpoint '{}'", endpoint, e);
```

Comment:
Why it's debug level?

Reply:
sorry, it is a mistake. changed to error